docs/pages/app-signing/security.mdx

```diff
@@ -96,6 +112,42 @@ For ad-hoc builds, if a user were to gain access to your session token it would
 
 None. They are available through the Apple Developer console.
 
+## iOS submit credentials
+
+### Apple App Store Connect (ASC) API key
+
+Apple App Store Connect (ASC) API key is one of the authentication methods that can be used to submit an iOS app to Apple's App Store using the EAS Submit service. This key is permanently stored on the Expo servers (unless deleted by a developer) while being encrypted using [KMS](https://cloud.google.com/security/products/security-key-management) when at rest.
```

Comment:
```suggestion
Apple App Store Connect (ASC) API key is one of the authentication methods that can be used to submit an iOS app to Apple's App Store using the EAS Submit service. This key is stored on the Expo servers and encrypted using [KMS](https://cloud.google.com/security/products/security-key-management) when at rest. The key will remain on Expo servers to be re-used for subsequent submissions, and it can be removed at any time by a user with the requisite permissions.
```